laravel-app/app/Http/Controllers/ProfileController.php

```diff
@@ -69,9 +58,22 @@ public function update(UpdateProfileRequest $request): RedirectResponse
     /**
      * Show specific user profile.
      */
-    public function show(int $userId): View
+    public function show(int $userId): mixed
```

Comment:
why mixed?

Reply:
+1

Reply:
pag View po kasi sir, nag eerror sa larastan, minsan po kasi RedirectResponse yung return pag empty or View kapag meron

Reply:
You can define the type more properly think about it.

Reply:
done sir, used union type View|RedirectResponse to specify return type. Thank you ❤️